### PR TITLE
Got the category list alphabetized

### LINKED ISSUE #39: View All Categories
As an admin I would like to see all the available categories so that I can choose to edit or delete one, or see that I should add a new one.

**Given** an admin is in the app  
**When** they select the `Category Management` link in the menu  
**Then** they should be directed to a page that lists all the Category names ordered alphabetically  

> **NOTE:** For the time being it is acceptable to treat all users as `admin` users. There is a future story about enforcing user permissions.

### PR DESCRIPTION
**Pull request by Jesier**
closes #39 
Got the category list alphabetized

### CODE REVIEW

File: Tabloid/Properties/launchSettings.json [hunk 1]
```diff
@@ -1,13 +1,4 @@
 {
```
Comment (dalamcd) [inline Tabloid/Properties/launchSettings.json]: This file should not be included in pull requests

File: Tabloid/Controllers/CategoryController.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Tabloid.Repositories;
+
+namespace Tabloid.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CategoryController : ControllerBase
```
Comment (dalamcd) [inline Tabloid/Controllers/CategoryController.cs:11]: Need to add [Authorize] above the controller class definition to protect routes from unauthorized users

File: Tabloid/client/src/modules/categoryManager.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const baseUrl = '/api/category';
+
```
Comment (dalamcd) [inline Tabloid/client/src/modules/categoryManager.js:3]: Need to use the getToken() method in authManager.js to get the currently logged in user's token, and then send that to the server in the Authorization header as part of the fetch call (see _doesUserExist() and _saveUser() in authManager.js for an example of how to get the token and how to send it)

**Review by dalamcd — CHANGES_REQUESTED** [review]

File: Tabloid/client/src/modules/categoryManager.js (hunk 3 above)
Comment (dalamcd) [reply Tabloid/client/src/modules/categoryManager.js:3]: This is the same for all the fetch calls, by the way, not just the one this comment is above!

File: Tabloid/Properties/launchSettings.json (hunk 1 above)
Comment (Jesier) [reply Tabloid/Properties/launchSettings.json]: I thought the git ignore did this was it not supposed to?

**Comment by Jesier** [comment]
change requests done

**Review by dalamcd — APPROVED** [review]
Everything looks good!

**Pull request merged**